### PR TITLE
Person

### PR DESCRIPTION
**Pull request by osbaldoAguilar**
Added media query and layout

### CODE REVIEW

File: htdocs/themes/berlin_mcjc/people/items/show.php [hunk 1]
```diff
@@ -19,8 +19,9 @@
 }
 ?>
 <div class="primary person<?php echo $itemClasses; ?>">
+<div class="person title">
```
Comment (timstallmann) [inline htdocs/themes/berlin_mcjc/people/items/show.php:22]: does this need to be in a div or can the class just go right on the <h1> element?

File: htdocs/themes/berlin_mcjc/people/common/person-page-oral-history-item.php [hunk 2]
```diff
@@ -6,12 +6,12 @@
 
   <!-- Item files -->
   <?php if (metadata($item, 'has files')): ?>
-    <div id="player" class="element">
+    <div class="player-element">
```
Comment (timstallmann) [inline htdocs/themes/berlin_mcjc/people/common/person-page-oral-history-item.php:9]: should this be `player element`?

**Comment by osbaldoAguilar** [comment]
Alright I put the class in the h1

**Comment by osbaldoAguilar** [comment]
And by this you mean that the class should be “player element” with a space
instead of a hyphen cause I did that as well

**Pull request merged**